awx/api/serializers.py

```diff
@@ -1709,8 +1704,17 @@ def get_related(self, obj):
                 labels=self.reverse('api:inventory_label_list', kwargs={'pk': obj.pk}),
             )
         )
+        if obj.kind in ('', 'constructed'):
```

Comment:
if these are ones that shouldn't show for "smart" inventory should we make this if statement reflect the condition?

Reply:
you mean change this to `obj.kind != 'smart'`?